exercises/meetup/canonical-data.json

```diff
@@ -3,446 +3,567 @@
+        {
+          "description": "saturteenth",
+          "cases":[
+            {
+              "description": "testcase 1 : February 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 2,
+              "week": "teenth",
+              "dayofweek": "Saturday",
+              "dayofmonth": 16
+            },
+            {
+              "description": "testcase 2 : April 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 4,
+              "week": "teenth",
+              "dayofweek": "Saturday",
+              "dayofmonth": 13
+            },
+            {
+              "description": "testcase 3 : October 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 10,
+              "week": "teenth",
+              "dayofweek": "Saturday",
+              "dayofmonth": 19
+            }
+          ]
+        },
+        {
+          "description": "sunteenth",
+          "cases":[
+            {
+              "description": "testcase 1 : May 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 5,
+              "week": "teenth",
+              "dayofweek": "Sunday",
+              "dayofmonth": 19
+            },
+            {
+              "description": "testcase 2 : June 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 6,
+              "week": "teenth",
+              "dayofweek": "Sunday",
+              "dayofmonth": 16
+            },
+            {
+              "description": "testcase 3 : October 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 10,
+              "week": "teenth",
+              "dayofweek": "Sunday",
+              "dayofmonth": 13
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "description": "test first -day",
+      "cases":[
+        {
+          "description": "first Monday",
+          "cases":[
+            {
+              "description": "testcase 1 : March 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 3,
+              "week": "first",
+              "dayofweek": "Monday",
+              "dayofmonth": 4
+            },
+            {
+              "description": "testcase 2 : April 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 4,
+              "week": "first",
+              "dayofweek": "Monday",
+              "dayofmonth": 1
+            }
+          ]
+        },
+        {
+          "description": "first Tuesday",
+          "cases":[
+            {
+              "description": "testcase 1 : May 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 5,
+              "week": "first",
+              "dayofweek": "Tuesday",
+              "dayofmonth": 7
```

Comment:
It can be constructed, but none of these values match the input format in the description:

> the first Monday of January 2017

It may be that the input should look something like

```
"property": "date",
"input": "First Monday of January 2017",
"expected": { "year": 2017, "month": 1, "day": 2 }
```